docs/dgx-pod.md

```diff
@@ -260,37 +263,75 @@ ansible dgx-servers -k -a 'hostname'
 
 ### 6. Deploy Kubernetes to the compute node(s)
 
-1. Run the same `k8s-cluster` ansible-playbook, but this time, do not limit it to just the management node(s):
+1. Run the same `k8s-cluster` ansible-playbook, but this time, we run against both the managment nodes (kube-master) and GPU nodes GPU nodes (kube-node).
+
+
+```sh
+
+ansible-playbook -l k8s-cluster playbooks/k8s-cluster.yml
+
+
+# NOTE: If SSH requires a password, add: `-k`
+# NOTE: If sudo on remote machine requires a password, add: `-K`
+# NOTE: If SSH user is different than current user, add: `-u ubuntu`
+
+ansible-playbook -l k8s-cluster -k -u dgxuser -K playbooks/k8s-cluster.yml
+
+```
 
-   ```sh
-   # NOTE: If SSH requires a password, add: `-k`
-   # NOTE: If sudo on remote machine requires a password, add: `-K`
-   # NOTE: If SSH user is different than current user, add: `-u ubuntu`
-   ansible-playbook -i config/inventory -b playbooks/k8s-cluster.yml
-   ```
 
 2. Verify that the Kubernetes cluster is running. 
 
    ```sh
-   # You may need to manually run: `sudo cp ./k8s-config/artifacts/kubectl /usr/local/bin`
-   kubectl get nodes
+# You may need to manually run: `sudo cp ./k8s-config/artifacts/kubectl /usr/local/bin`
```

Comment:
Needs spaces - this one is extra important since it's causing a break in formatting. "# You may need..." is getting interpreted as a header instead of part of the code block.